core/sql/executor/ExExeUtilGet.cpp

```diff
@@ -7733,26 +7733,12 @@ else
     return rc;  
                 
   //EOD of LOB data file
-  
-  hdfsFS fs = currContext->getHdfsServerConnection((char*)getLItdb().getHdfsServer(),getLItdb().getHdfsPort());
-  if (fs == NULL)
-    return LOB_DATA_FILE_OPEN_ERROR;
-
-  
   snprintf(lobDataFilePath, LOBINFO_MAX_FILE_LEN, "%s/%s", lobLocation, lobDataFile);
-  hdfsFile fdData = hdfsOpenFile(fs, lobDataFilePath,O_RDONLY,0,0,0);
-  if (!fdData) 
-    {
-      hdfsCloseFile(fs,fdData);
-      fdData = NULL;
-      return LOB_DATA_FILE_OPEN_ERROR;
-    }
-  hdfsFileInfo *fInfo = hdfsGetPathInfo(fs, lobDataFilePath);
-  if (fInfo)
-    lobEOD = fInfo->mSize;
-  else
-    lobEOD = 0;
-  
+  HDFS_Client_RetCode hdfsClientRetcode;
```

Comment:
The toggle is missing here ? We shoudl preserve the old interface here.

Reply:
Yes. At places I have avoided useLibHdfs concept because it is confined and it can be safely replaced without any effect on lob access.